docs/im/ism/api.md

```diff
@@ -161,6 +161,10 @@ POST _opendistro/_ism/add/index_1
 }
 ```
 
+If you use a wildcard `*` while adding a policy to an index, the ISM plugin interprets `*` as all indices, including the `.opendistro-security` system index that stores user roles and tenants. A delete action in your policy might accidentally delete all user roles and tenants in your cluster.
```

Comment:
Could this also delete your `.kibana*` indices, your alerting settings index, etc.? Maybe something like "including system indices like `.opendistro-security`, which stores users, roles, and tenants."